performance/font-loading.md

```diff
@@ -0,0 +1,56 @@
+# Font Loading
+
+## Are you sure you need to use webfonts?
+
+It's worth asking this question initially. What benefit do you believe a webfont will give you and is it worth the cost in page performance? Afterall, system fonts are battle tested and were produced with legibility in mind.
+
+## Implementation
+We do not prescribe a particular implementation. Each team should determine which implementation is best suited for them. However, before selecting an implementation a team should be mindful of our working practices such as:
+
+- [Graded Browser Support](https://github.com/springernature/frontend-playbook/blob/main/practices/graded-browser-support.md)
+  - Is the implementation supported by all "enhanced" browsers? If not, is there a good justification for selecting it anyway?
+  - Does the implementation work in conjunction with the Cut The Mustard technique?
+- [Progressive Enhancement](https://github.com/springernature/frontend-playbook/blob/main/practices/progressive-enhancement.md)
+  - This is not meant to force you to choose a non-js implementation, but rather for you to ensure you consider what will happen for users in varying circumstances from the beginning.
+- [Our Performance Goals](https://github.com/springernature/frontend-playbook/blob/main/performance/performance-checklist.md)
+  - What impact will the implementation have on the page performance of your site? 
+  
+## Quick Tips
+Here are some quick suggestions for techniques that may prove useful:
+
+### Understand the different user experiences in font loading: FOIT, FOUT and FOIT
+
+- FOIT means flash of invisible text. When web fonts are loading, we hide text so users don’t see anything. We only show the text when web fonts are loaded.
```

Comment:
> When web fonts are loading, we hide text so users don’t see anything.

Maybe stick a "while" in there, in case people less familiar with technical details end up on this page, something like:

> Text on the page is hidden while webfonts are loading, so users will (or may??) see a page with images but no text. We only show the text when [...]